velox/docs/functions/presto/string.rst

```diff
@@ -153,6 +153,8 @@ String Functions
     Returns the position of the N-th ``instance`` of ``substring`` in ``string``.
     ``instance`` must be a positive number.
     Positions start with ``1``. If not found, ``0`` is returned.
```

Comment:
i dont think users know what is nextStartPosition, maybe just say that the function considers overlapped strings when counting instances.

Reply:
Updated.